packages/moonstone/ToggleIconItem/ToggleIconItem.js

```diff
@@ -0,0 +1,119 @@
+import kind from '@enact/core/kind';
```

Comment:
I'm not sure creating a new component is necessary for what we're trying to do. I also don't seem to be able to locate where it is used in the framework. I assume it's something that an App would have to import and use, rather than just passing in an existing icon component.

Reply:
I've removed new component `ToggleIconItem`. As you commented, applications do not need to select type of ToggleItem.